tests/scripts/validate_response.py

```diff
@@ -148,10 +160,11 @@ def main():
     result_df = ResponseValidation().get_response_quality(args, qa_pairs, api_client)
 
     if len(result_df) > 0:
-        result_dir = "tests/test_results"
+        result_dir = args.out_dir
         os.makedirs(result_dir, exist_ok=True)
         result_file = (
-            f"{result_dir}/question_answer_result_{args.scenario}_{args.model}.csv"
+            f"{result_dir}/question_answer_result_"
+            f"{args.scenario}_{args.model}_{args.provider}.csv"
```

Comment:
let's try to be consistent in naming ordering:  provider, then model (least specific to most specific)